website/addons/owncloud/static/owncloudLogActionList.json

```diff
@@ -0,0 +1,4 @@
+{
+    "owncloud_folder_selected" : "${user} selected ownCloud folder ${folder} in ${node}",
```

Comment:
This file should have each of the `<addon>_<action>` logs common to storage providers with a mutable file/folder structure. This includes:
```
_file_added
_file_removed
_file_updated
_folder_created
_folder_selected
_node_authorized
_node_deauthorized
_node_deauthorized_no_user
```